book/src/concepts/state.md

```diff
@@ -224,7 +224,7 @@ impl State<(), MyEvent> for GameplayState {
 # fn main() {}
 ```
 
-To make `Application` aware of the change to which events to send to the state, you also need to supply both the 
+To make `Application` aware of the change, and decide to which events the state should be send, you need to supply both the 
```

Comment:
```suggestion
 To make `Application` aware of the change, and decide which events to send to the state, you need to supply both the 
```

Reply:
This GitHub-centric patch-on-a-patch-on-a-patch workflow is new to me. I hope I won't blow it all up.